packages/concerto-core/test/introspect/introspector.js

```diff
@@ -80,6 +81,32 @@ describe('Introspector', () => {
             const introspector = new Introspector(modelManager);
             introspector.getClassDeclaration('org.acme.base.Person').should.not.be.null;
         });
+
+        it('should be able to handle the aliased imported types', () => {
+            // create and populate the ModelManager with a model file
+            const modelManager = new ModelManager();
+            Util.addComposerModel(modelManager);
+            modelManager.enableAliasedType = true;
```

Comment:
"enable" is a verb, so I'd expect `enableAliasedType` this to be a method call. It would be better to pass this option to the ModelManager constructor, something like:

new ModelManager( { importAliasing: true } )

Reply:
Actually we have a field `enableMapType` which existed earlier. I have used the same convention in the baseModelManager.

Yes we can pass the field in the constructor

Reply:
renamed variable to importAliasing